vowpalwabbit/cb_explore_adf_regcb.cc

```diff
@@ -279,7 +280,7 @@ base_learner* setup(VW::setup_base_i& stack_builder)
                .one_of({"mtr"})
                .help("Contextual bandit method to use. RegCB only supports supervised regression (mtr)"));
 
-  options.add_and_parse(new_options);
+  options.add_parse_and_check_necessary(new_options);
```

Comment:
this might be incorrect bc of line 285

Reply:
Hmm good point. So it is `cb_explore_adf` and `regcb` or `regcbopt` right? How should we handle that?